admin_guide/topics/creating-primed-images-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='creating-primed-image-cluster-auto-scaler-{context}']
+= Creating a primed image
+
+You can use Ansible playbooks to automatically create a
+primed image for the auto-scaler to use. You must provide attributes
+from your existing Amazon Web Services (AWS) cluster.
+
+[NOTE]
+====
+If you already have a primed image, you can use it instead of creating a new one.
+====
+
+.Procedure
+
+On the host that you used to create your {product-title} cluster, create a
+primed image:
+
+. Create a new Ansible inventory file on your local host:
++
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+ifdef::openshift-enterprise[]
+openshift_deployment_type: openshift-enterprise
```

Comment:
This is actually wrong. I was right before, and wrong to change it so many times. Sorry!

Inventory entries are separated by ':'. The extra provisioning files are YAML and are separated by '='.

This entry should be: `openshift_deployment_type=openshift-enterprise`.

Reply:
Ah! Thank you for explaining the difference between the two references! I own my part in this - I was resetting them both to be the same. :/